jobs/JEVS_RTOFS_PLOTS

```diff
@@ -36,7 +36,7 @@ cd $DATA
 export NET=${NET:-evs}
 export STEP=${STEP:-plots}
 export COMPONENT=${COMPONENT:-rtofs}
-export RUN=${RUN:-argo}
+export RUN=${RUN:-ocean}
```

Comment:
Can we can `OBTYPE=${OBTYPE:-argo}` or some similar default value for OBTYPE?

Reply:
@malloryprow, Yeah, I noticed that for plots j-job this variable had been defined with default value "argo" but I would like to know more about the background and the reason for that, so, I will have a better idea for this change. Thanks!

Reply:
Sure! I believe this relates to how NCO wants variables like this all have a default setting. I'd say especially so since `OBTYPE` is used to call the ex-script. If `OBTYPE` wasn't defined in the ecf or driver script for some reason when it would get to `$HOMEevs/scripts/$STEP/$COMPONENT/exevs_${COMPONENT}_${OBTYPE}_${VERIF_CASE}_${STEP}.sh` it wouldn't be correct.

I may be remembering this wrong. @AliciaBentley-NOAA @PerryShafran-NOAA @ShelleyMelchior-NOAA Thoughts?

Reply:
That's correct. NCO would like us to give variables a default value in the J-jobs incase that variable isn't defined upstream like it should be. Defining a default value in the J-jobs also lets NCO know that a variable SHOULD be defined upstream. It's basically good bookkeeping for NCO. 

Now that both the RTOFS .ecf scripts and dev drivers have been updated to include an OBTYPE variable, OBTYPE will always be set upstream of the J-jobs and the default value in the J-jobs will always be overwritten. Please let us know if that doesn't make sense!

@malloryprow @SamiraArdani-NOAA @PerryShafran-NOAA @ShelleyMelchior-NOAA

Reply:
@malloryprow and @AliciaBentley-NOAA, Correct. I understand. Just to be clear, I was not sure why in the plots j-job, this line below, has been defined with -argo value. It has now changed in this PR but it has been always my question:
https://github.com/SamiraArdani-NOAA/EVS/blob/develop/jobs/JEVS_RTOFS_PLOTS#L39

Reply:
@malloryprow and @AliciaBentley-NOAA, Oh, I think I understand now. I am adding the default value.